csclient/params/helpers.go

```diff
@@ -0,0 +1,70 @@
+// Copyright 2016 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package params
+
+import (
+	"gopkg.in/juju/charm.v6-unstable/resource"
+)
+
+// Resource2API converts a charm resource into an API Resource struct.
+func Resource2API(res resource.Resource) Resource {
```

Comment:
I'm not convinced this is the right package for the conversion helper. I think it's more appropriate in csclient or even the root charmrepo package.  I think the whole point is for the params package to only know or care about API params.